### ISSUE TITLE
Searching Segfault

### ISSUE THREAD

**Issue by barrett-ruth** [ahrm/sioyek#1277]
Open [this](https://www.usenix.org/system/files/conference/nsdi12/nsdi12-final138.pdf) pdf.

Search "RAM".

Crash.
![image](https://github.com/user-attachments/assets/b5329bc5-1af7-47cf-90eb-5e07b6393e0d)

**Comment by ahrm** [comment]
Can't reproduce the issue. Which platform are you on and which sioyek version are you using? Does the issue happen with the preview release here https://github.com/ahrm/sioyek/releases/tag/sioyek3-alpha0?

**Comment by barrett-ruth** [comment]
Yeah. The maintainer of the AUR package just does not know how to keep a package updated. Sorry - all on my end

**Comment by ahrm** [comment]
No this is not the maintainer's fault. This is an experimental build so I think they are waiting for the final release.
